bin/MadGraph5_aMCatNLO/cards/production/13TeV/VLferm_EWcouplings_4321/VLferm_EWcouplings_4321_m900GeV_EE_to_4b_proc_card.dat

```diff
@@ -0,0 +1,6 @@
+import model VLferm_UFO
```

Comment:
Is the model 4 FS or 5FS?

Reply:
The model is in 5FS. In this case we are only considering EW production of the VLLs  the initial state b's don't really have an impact, since it process through s-channel Z/gamma and W  (though still they are included in the proton definition and we use massless b). However, for other states within the model, due to the highly generation-dependent structure, the initial state b's would be a dominant contribution.